Projekty/Projekt1/AdriannaGrudzień_KingaUłasik/WUM_projekt1_inżynieria_cech.ipynb

```diff
@@ -0,0 +1,1810 @@
+{
```

Comment:
<p>Wiem, że o tym rozmawialiśmy i wiem, że wiecie o co chodzi, dlatego proszę poprawcie to, żeby było przyzwoicie.</p><p> </p><p>random_state nie może być optymalny i nie jest parametrem</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/62/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>